runtime/doc/lsp.txt

```diff
@@ -1942,12 +1942,10 @@ make_given_range_params({start_pos}, {end_pos}, {bufnr}, {offset_encoding})
                            selection.
       • {bufnr}            (`integer?`) buffer handle or 0 for current,
                            defaults to current
-      • {offset_encoding}  (`'utf-8'|'utf-16'|'utf-32'?`) defaults to
-                           `offset_encoding` of first client of `bufnr`
+      • {offset_encoding}  (`'utf-8'|'utf-16'|'utf-32'`)
```

Comment:
>  Rename `offset_encoding` to `position_encoding`?

These all look like positional parameters so that definitely makes sense (maybe as a separate PR), if that aligns with the LSP spec name.

Reply:
> If there is some reason to avoid that change, at least mentioning "aka position encoding" in the docs is helpful.

While the parameters to the `make_*` functions are positional, in 99% of the call-sites we pass `client.offset_encoding`. The names should imho match.
Not sure if it's warranted to rename the client's property. I'd tend not to.